lib/chefs-de-file/routes.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express')
+const createError = require('http-errors')
+const {isEqual} = require('lodash')
+const errorHandler = require('../util/error-handler')
+const {ensureIsAdmin} = require('../util/middlewares')
+const w = require('../util/w')
+const ChefDeFile = require('./model')
+
+async function chefsDeFileRoutes() {
+  const app = new express.Router()
+
+  app.use(express.json())
+
+  app.param('chefDeFileId', w(async (req, res, next) => {
+    const {chefDeFileId} = req.params
+    const chefDeFile = await ChefDeFile.fetch(chefDeFileId)
+
+    if (!chefDeFile) {
+      throw createError(404, 'L’identifiant de chefDeFile demandé n’existe pas')
+    }
+
+    req.chefDeFile = chefDeFile
+    next()
+  }))
+
+  app.route('/chefs-de-file/:chefDeFileId')
+    .get(ensureIsAdmin, w(async (req, res) => {
+      res.send(req.chefDeFile)
+    }))
+    .put(ensureIsAdmin, w(async (req, res) => {
+      const chefDeFile = await ChefDeFile.update(req.chefDeFile._id, req.body)
+
+      if (isEqual(req.chefDeFile, chefDeFile)) {
```

Comment:
La 304 ne sert pas à ça.
Ça n'est pas un soucis si le PUT ne change rien à l'objet in fine, tu peux enlever complètement le test isEqual.

Reply:
304 c'est ce qu'un serveur renvoie sur un GET pour dire "les données sont identiques par rapport à ce que tu as en cache"

Reply:
Ok c'est ce comportement que je cherchais à imiter, mais sans savoir qu'il se limitait au `GET`.